README.md

```diff
@@ -3,154 +3,110 @@ This repository holds the kcp tests that tests against the publicly available  i
+"[area/workspaces] Author:pewang-Medium-[Smoke] Multi levels workspaces lifecycle should work [Suite:kcp/smoke/parallel/minimal]"
+"[area/workspaces] Author:zxiao-Medium-[Serial] I can create context for a specific workspace and use it [Suite:kcp/smoke/serial]"
 ...
 ```
 You can save the above output to a file and run it:
 ```console
 $ ./bin/extended-platform-tests run -f <your file path/name>
 ```
-If you want to run a test case, such as `g.It("Author:jiazha-Critical-23440-can subscribe to the etcd operator  [Serial]"`, since the `TestCaseID` is unique, you can do:
+##### Run all smoke automation test cases
+If you want to run all smoke test cases which has the `[Smoke]` label, you can do:
 ```console
-$ ./bin/extended-platform-tests run all --dry-run|grep "23440"|./bin/extended-platform-tests run --junit-dir=./ -f -
+$ ./bin/kcp-tests run smoke
+```
+##### Run a single automation test case
+If you want to run a single test case, such as `g.It("Author:pewang-Medium-[Smoke] Multi levels workspaces lifecycle should work"`, you can do:
+```console
+$ ./bin/kcp-tests run all --dry-run|grep "Multi levels workspaces lifecycle should work"|./bin/kcp-tests run --junit-dir=./ -f -
 ```
 
 ### Debugging
-#### Keep generated temporary project
-Sometime, we want to **keep the generated namespace for debugging**. Just add the Env Var: `export DELETE_NAMESPACE=false`. These random namespaces will be kept, like below:
+#### Keep generated temporary workspaces
+Sometime, we want to **keep the generated workspaces for debugging**, we could just set **`export DELETE_WORKSPACE=false`**, then these temporary workspaces will be kept. 
+
 ```console
 ...
 Dec 18 09:39:33.448: INFO: Running AfterSuite actions on all nodes
-Dec 18 09:39:33.448: INFO: Waiting up to 7m0s for all (but 100) nodes to be ready
-Dec 18 09:39:33.511: INFO: Found DeleteNamespace=false, skipping namespace deletion!
-Dec 18 09:39:33.511: INFO: Running AfterSuite actions on node 1
 ...
-1 pass, 0 skip (2m50s)
-[root@preserve-olm-env openshift-tests-private]# oc get ns
-NAME                                               STATUS   AGE
-default                                            Active   4h46m
-e2e-test-olm-a-a92jyymd-lmgj6                      Active   4m28s
-e2e-test-olm-a-a92jyymd-pr8hx                      Active   4m29s
+1 pass, 0 skip (1m50s)
+$ kubectl get ws
+NAME                        TYPE        PHASE   URL
+e2e-test-kcp-syncer-s6ktv   universal   Ready   https://<kcp-test-env-domain>/clusters/root:users:rp:pv:rh-sso-xxxx:e2e-test-kcp-syncer-s6ktv
+e2e-test-kcp-syncer-a5spq   universal   Ready   https://<kcp-test-env-domain>/clusters/root:users:rp:pv:rh-sso-xxxx:e2e-test-kcp-syncer-a5spq
 ...
 ```
 #### Print cluster event on Terminal
-When you execute cases, there are some cluster event which is printed to the terminal, like
+When you execute cases, there are some events which is printed to the terminal (**`currently we cannot retreive events from kcp`)**, like
```

Comment:
Given we currently can't, how about adding TODO after 108 and commenting out all lines after 108? And if in future we can, we can remove the comments.

Reply:
Sure, added.